cloud/passportauth/src/user/BaseUser.ts

```diff
@@ -0,0 +1,32 @@
+export interface BaseUser {
```

Comment:
See the current [BaseTask](https://github.com/feedhenry-raincatcher/raincatcher-core/blob/vehicle-inspection-example/cloud/wfm/src/task/BaseTask.ts) for an example, it implements the `Task` interface just with the intent of making it easier for users to override the parts they need to.